modules/electrophysiology_browser/php/electrophysiologybrowserrow.class.inc

```diff
@@ -37,10 +38,11 @@ class ElectrophysiologyBrowserRow implements \LORIS\Data\DataInstance
      *                     returns
      * @param integer $cid The centerID affiliated with this row.
```

Comment:
I think you'll need to add a PHPDoc here for the new param or else Travis will complain.